.github/workflows/integration-test.yml

```diff
@@ -13,6 +13,7 @@ on:
       - '.github/**'
       - '!.github/workflows/integration-test.yml'
   workflow_dispatch:
+  workflow_call:
```

Comment:
What are we using workflow_call for?

Reply:
this allows the upload-release workflow to call the integ test as a pre-step to upload

Reply:
Oh right. Reusable workflows.